test/helpers/getCodeFromSass.js

```diff
@@ -1,10 +1,8 @@
 import path from "path";
 import fs from "fs";
 
-import { klona } from "klona/full";
-
 function getCodeFromSass(testId, options) {
-  const loaderOptions = klona(options);
+  const loaderOptions = { ...options };
```

Comment:
I beleive we should keep `klona` becasue it will create a deep copy whereas `destructring` will create a shallow copy which might be a problem for nested options.

Reply:
Yes, I do understand it but @alexander-akait wanted to try removing klona and replace it with de structuring. That's why raised a PR

Reply:
@harish-sethuraman can you fix conflicts, so that the tests can run?